cfme/tests/physical_infrastructure/test_physical_server_details.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils import testgen
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3)]
+
+pytest_generate_tests = testgen.generate([LenovoProvider], scope="module")
```

Comment:
Please use env marker here instead of testgen.generate, will have same effect.

Add to `pytestmark` list on L9 above:
`pytest.mark.provider([LenovoProvider], scope="module")`